embedding_matrix_api/app/settings.py

```diff
@@ -0,0 +1,36 @@
+"""Settings module."""
+
+from pathlib import Path
+
+from pydantic import BaseSettings
+
+
+class APISettings(BaseSettings):
+    """Settings class.
+
+    Set variables to be used
+    """
+
+    API_V1_STR: str = "/api/v1"
+    PROJECT_NAME: str = "ML inference service"
```

Comment:
@vicente87 This is not the ML inference service.

Reply:
@jaime-cespedes-sisniega I already modified this